src/transaction_manager/transaction.cpp

```diff
@@ -104,9 +104,9 @@ addr_t const &Transaction::getSender() const {
                          "\nOriginal RLP: " + (cached_rlp_ ? dev::toJS(*cached_rlp_) : "wasn't created from rlp"));
 }
 
-template <bool for_signature>
+template <bool for_signature, bool w_sender>
```

Comment:
Note: We might differentiate between standard parameters and template parameters by having template parameters names uppercase template <bool FOR_SIGNATURE, bool W_SENDER>, but not a requirement :)